nav2_costmap_2d/test/integration/inflation_tests.cpp

```diff
@@ -600,3 +601,48 @@ TEST_F(TestNode, testInflation3)
   ASSERT_EQ(countValues(*costmap, nav2_costmap_2d::LETHAL_OBSTACLE), 1u);
   ASSERT_EQ(countValues(*costmap, nav2_costmap_2d::INSCRIBED_INFLATED_OBSTACLE), 4u);
 }
+
+
+/**
+ * Test dynamic parameter setting of inflation layer
+ */
+TEST_F(TestNode, testDynParamsSet)
+{
+  auto node = std::make_shared<rclcpp_lifecycle::LifecycleNode>("inflation_dyn_param_test");
```

Comment:
This parameter is unused. Why do we need to keep it?

Reply:
Aaah, thanks great spot!